Tests/WorkspaceTests/AuthorizationProviderTests.swift

```diff
@@ -9,14 +9,16 @@
 // See http://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
 //
 //===----------------------------------------------------------------------===//
+import Foundation
 
 @testable import Basics
 import _InternalTestSupport
 import Workspace
-import XCTest
+import Testing
 
-final class AuthorizationProviderTests: XCTestCase {
-    func testNetrcAuthorizationProviders() throws {
+struct AuthorizationProviderTests {
```

Comment:
I suggest making all newly added test structs `fileprivate` to avoid exporting unnecessary symbols (and make it clear to readers that the code is entirely localized).

Reply:
done